### ISSUE TITLE
SSH to device without agent (via relay) does not work if user doesn't have full right access on the group of the relay

### ISSUE THREAD

**Issue by iez0** [Ylianst/MeshCentral#4682]
**Describe the bug**
The users that doesn't have full rights on the group of the agent that serve as a relay cannot ssh into relayed devices. 

**To Reproduce**
Meshcentral is in wan or hybrid mode, 2 accounts, one is admin called A, the other is a user called U.
"A" create an agent device group called G with one device, and a device group "no agent devices relayed thow agent" called R with one linux device L.
If the account U have full right on G, it can ssh to L.
if the account U doesn't have full right on G, the "SSH connect" in the device L does nothing. No popup to type user and password. Just a small flash of the terminal with a rectangle caret on the upper left corner.
Giving full right on R to U does not change the behaviour.
Giving all rights on G to U except full right (Edit Group, Manage Users, Manage Devices, Control, Console, Server Files, Wake, Notes, Chat, Uninstall, Commands, Reset/Off) does not change the behaviour, U can't ssh to L.

**Expected behavior**
Users should be be able to ssh into hosts without "full rights" on the device group of the relay.

**Server Software:**
 - OS: alpine linux
 - Virtualization: VMM (OpenBSD) or Hyper-V (Windows 2019)
 - Network: WAN mode or Hybrid mode
 - Version: 1.0.91
 - Node: 16.17.1

**Client Device:**
 - Device: laptop
 - OS: OpenBSD or Windows
 - Network: Local or Remote
 - Browser: Firefox

**Relay Device:**
 - Device: any. I tried a remote debian linux server, or a remote windows 10 host, or the meshcentral server with agent installed (as a minimal vm to reproduce the bug).
 - OS: alpine linux 3.16, linux debian 11, windows 10 pro 21h2
 - Current Core Version : Aug 29 2022, 2893861893 (debian) or Current Core: Aug 25 2022, 461107747 (windows)

**Remote Device:**
 - Any, doesn't event try to connect

**Your config.json file**
```
{
  "$schema": "http://info.meshcentral.com/downloads/meshcentral-config-schema.json",
  "settings": {
    "cert": "100.64.4.3"
  },
  "domains": {
    "": {
      "ssh": true
    }
  }
}
```

BTW thanks for this great piece of software ! It's very usefull in our job.

**Comment by Ylianst** [comment]
I just push in a fix for this, will be in MeshCentral v1.0.95. I added a new "Relay" permission that allows a user to relay thru a device. Note that "Control" permission also allows relay.

![image](https://user-images.githubusercontent.com/1319013/199359123-efdee6d5-420c-4415-a699-69e0837cb785.png)

Once updated, let me know if it works.

**Comment by iez0** [comment]
It works! Thank you!
